OpenRA.Game/Map/ActorInitializer.cs

```diff
@@ -14,10 +14,26 @@
 
 namespace OpenRA
 {
-	public class ActorInitializer
+	public interface IActorInitializer
```

Comment:
This one too, unless you did have a plan for these ones that you've kept around.